data/flatpak_updater.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
I think that `support/flatpak` would be a more natural location for these files.

When is data/flatpak_updater.sh supposed to be run. It seems unused in this version of the change?

Reply:
Make sense, moved.